### ISSUE TITLE
Peer Review

### ISSUE THREAD

**Issue by kylewschulz** [SkyWang0919/STATS-506-Midterm-Project-02#3]
Hello, 

Here are the results of my peer review... 

**_Core example (Python):_**

- First glance is that python code looks solid 

- You mentioned in your to-do-list that inline comments are still needed, which I agree with, a little hard to follow without the inline comments. 

- First visualization could be helped by including a title 

- The tables could look better with better column names, currently just have the variable names from code, not super presentation-like. 

**_Common Sections:_**

- Report has all sections needed. 

- Hyperlinked sections help navigation to different parts. 

- Dataset is explained well and good explanation of motivation. 

- To do list exists in report, but may be helpful to also include in the github/readme somewhere so you don't need to download and launch a file to access. 

**_Git:_** 

- Readme exists. Easy to tell who is responsible for each language of code. Provides easy access to data files and code needed for the tutorial. 

- Nitpicky, but I believe it would help/is suggested if you have your individual github's linked in the readme. 

- There is evidence of all members pushing to the git repo. Commit comments show evidence of code review within group. 

- Very solid organization of files, right files are on the top level. Super super nitpicky, but maybe the python files could be named slightly better, wasn't 100% what the difference between Python-Part and interactive were without opening them. Might help to have explanations of every file in the readme. 

**_Presentation/Style:_** 

- Tutorial is looking really solid. As mentioned before, might help to have the table columns renamed to something less code-like. I really like the python visualizations. Guide is easy to follow. 

Good luck on the rest of your project! 

Kyle

**Comment by longyyu** [comment]
Thanks for the feedback!

We've added more inline comments. Links to members' individual repositories have also been included in README. 

As for the column names, we did not change them to be less code-like, because we want the readers to be able to link data with the code, so that when they look at something like `ggplot(aes(x = stringency_index, y = human_development_index, size = total_deaths_per_million, color = continent))`, they know what each of these columns looks like.
